css/index.css

```diff
@@ -1,3 +1,430 @@
+	margin: 15px;
+}
+
+.services .services-content .services-header-img {
+	margin-bottom: 25px;
+	width: 100%;
+}
+
+.services p {
+	font-size: 10px;
+}
+
+.services .services-text {
+	border-bottom: 1px solid darkgray;
+	padding-bottom: 25px;
+}
+
+.services .cards {
+	border-bottom: 1px solid darkgray;
+	display: flex;
+	justify-content: space-between;
+	flex-wrap: wrap;
+	margin: 25px 0;
+}
+
+.services .card {
+	background: whitesmoke;
+	border: 1px solid darkgray;
+	margin-bottom: 25px;
+	padding: 10px;
+	width: 48%;
+}
+
+.services .card button {
+	border-radius: 5px;
+	margin-top: 10px;
+}
+
+.services .services-info {
+	display: flex;
+	justify-content: space-between;
+	margin-bottom: 25px;
+}
+
+.services .facts {
+	width: 48%;
+}
+
+.services .facts ul {
+	list-style-type: disc;
+	list-style-position: inside;
+	margin-top: 10px;
+}
+
+.services .facts ul li {
+	margin: 5px;
+}
+
+.services .services-info-img {
+	height: auto;
+	width: 48%;
+}
+
+/* Vision page */
+
+.vision {
+	width: 70%;
+}
+
+.vision p {
+	font-size: 10px;
+}
+
+.vision header {
+	align-items: center;
+	display: flex;
+	justify-content: space-around;
+	margin: 20px 0;
+}
+
+.vision header nav {
+	display: flex;
+	justify-content: space-around;
+	margin: 15px;
+}
+
+.vision header nav a {
+	margin: 15px;
+}
+
+.vision header .logo {
+	height: 50px;
+	width: auto;
+}
+
+.vision .left {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	border-right: 1px solid darkgray;
+	padding: 20px;
+	margin: 20px;
+	width: 50%;
+}
+
+.vision .vision-content {
+	display: flex;
+}
+
+.vision .right {
+	width: 50%;
+}
+
+.vision .right-imgs img {
+	border-radius: 2px;
+	height: auto;
+	margin: 15px 0;
+	width: 100%;
+}
+
+.vision .right .right-head {
+	border-bottom: 1px solid darkgray;
+	padding-bottom: 15px;
+}
+
+/* tablet css */
+
+@media (max-width: 800px) {
+
+    .container {
+        margin-left: 0;
+        width: 100%;
+    }
+
+    nav {
+        flex-wrap: wrap;
+        justify-content: space-between;
+        margin: 0 0 20px 0;
+    }
+
+    nav .logo-container {
+        display: flex;
+        justify-content: center;
+        order: -1;
+        width: 100%;
+        margin: 0 0 44px;
+    }
+
+    .banner {
+        padding: 0;
+        margin: 0 0 33px 0;
+    } 
+
+    .banner img {
+        display: none;
+    }
+
+    .banner div {
+        text-align: center;
+    }
+
+    .banner h1 {
+        font-size: 70px;
+        margin: 0 0 28px 0;
+    }
+
+    .banner h4 {
+        padding: 8px 45px;
+    }
+    
+    .top-content {
+        margin: 0 0 46px 0;
+    }
+
+    .middle-img {
+        margin-bottom: 51px;
+    }
+
+    .bottom-content {
+        padding-bottom: 76px;
+        margin-bottom: 37px;
+    }
+
+    .blurb {
+        width: 45%;
+    }
+
+    .contact span {
+        margin: 0 0 15px 0;
+    }
+
+}
+
+/* mobile css */
+@media (max-width: 500px) {
+    
+    body {
+        background-color: bisque;
+    }
```

Comment:
I like changing the background color here so you can see the change. Even if it's just for development.